test/BUILD.gn

```diff
@@ -183,7 +183,6 @@ test("brave_unit_tests") {
     "//brave/browser/ui/brave_tooltips",
     "//brave/browser/ui/webui/new_tab_page:unit_tests",
     "//brave/browser/web_discovery:unit_tests",
-    "//brave/browser/widevine:unittest",
```

Comment:
that removal is valid. We have a correct dep that uses `if (enable_widevine)` below:
https://github.com/brave/brave-core/blob/437a66e377eb02b132b36e51bcd3a07b2a258569/test/BUILD.gn#L315-L317